File: server/src/game/tools/conf/output/ts/TipConfig.ts
class TipConfig {
    private static _dic:Dictionary<TipConfig> = new Dictionary<TipConfig>();
    private static _arr:Array<TipConfig> = new Array<TipConfig>();

	 public Id :number ;
     public Tip :string ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<TipConfig>{
        return  TipConfig._dic;
    }
    public static get Arr():Array<TipConfig>{
        return TipConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:TipConfig = <TipConfig>json[i];
            TipConfig._arr.push(config);
            TipConfig._dic.add(config.Id,config);
        }
    }
}

File: server/src/game/conf/TipConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type TipConfigElem struct {
	Id int
	Tip string
	
}

type TipConfig0 struct {
	Dic map[int]*TipConfigElem
	Arr []*TipConfigElem
}

var (
	TipConfig TipConfig0
)

func loadTipConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	TipConfig.Dic = make(map[int]*TipConfigElem)
	TipConfig.Arr = make([]*TipConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &TipConfig.Arr)
	for _,conf := range TipConfig.Arr{
		TipConfig.Dic[conf.Id] = conf
	}
	
}
